matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/crypto/verification/DefaultVerificationService.kt

```diff
@@ -457,7 +457,29 @@ internal class DefaultVerificationService @Inject constructor(
         Timber.d("## SAS onStartRequestReceived ${startReq.transactionId}")
         if (checkKeysAreDownloaded(otherUserId!!, startReq.fromDevice) != null) {
```

Comment:
It seems dangerous to force a nullable field.

Reply:
done